site/source/tutorials/006_deploying_to_production/index.md

```diff
@@ -0,0 +1,143 @@
+
+Since all of the files in the `dist` folder are optimized for use in production, deployment only involves copying the contents of the `dist` folder to the location that the web application expects them to be.
+
+While the default `dojo build` command can handle many common scenarios, there are some arguments that can be passed to the command to influence the build process. We will talk about those in the next section.
+
+{% section %}
+
+## Build options
+
+{% task 'Learn how to customize a Dojo 2 build.' %}
+
+The default options for `dojo build` are sufficient for many situations, but there are cases where additional configuration is required.
+
+{% instruction 'Run the `dojo build help` command and examine the result.' %}
+
+There are four groups of command line arguments that are available when using the `dojo build` command.
+
+The first group consists of the `--watch` and `--port` arguments. These are used to start and configure the development server that is designed to host Dojo 2 applications as they are being authored.
+
+The `--help` argument displays the help information for the `dojo build` command and is equivalent to the `dojo build help` command.
+
+{% aside 'I18n in Dojo 2' %}
+Dojo 2 has a complete set of internationalization (i18n) capabilities that are beyond the scope of this tutorial. To learn more about i18n in Dojo 2, refer to the [Internationalization in Dojo 2 tutorial](../comingsoon.html).
+{% endaside %}
+
+The `--locale`, `--supportedLocales`, and `--messageBundles` arguments are used to configure the build's support for internationalization. The `--locale` argument is used to specify which language should be considered the *default*. Whenever a different locale is used that does not have a specific translation, the default locale's translation will be used. The `--supportedLocales` argument is used to specify all of the locales that the application is designed to work with. Each locale that is added here will have its rules for date formatting, currency, etc. added to the build. Finally, the `--messageBundles` argument specifies all of the message bundles, or translation files, that should be included in the build. For more information about creating internationalized applications with Dojo 2, refer to the [Internationalization](../comingsoon.html) article in the reference guide.
+
+The next two arguments that are listed when the `dojo build help` command is run are the `--element` and `--elementPrefix`. These two fields are used when exporting Dojo 2 widgets as [web components](https://en.wikipedia.org/wiki/Web_Components). They are not used when building an application.
```

Comment:
These two fields are used when exporting Dojo 2 widgets as [web components](https://en.wikipedia.org/wiki/Web_Components). They are not used when building an application. -> These two fields are primarily used when exporting Dojo 2 widgets as [web components](https://en.wikipedia.org/wiki/Web_Components) rather than for building stand alone Dojo 2 applications.